### ISSUE TITLE
Elasticsearch Datasource

### ISSUE THREAD

**Issue by vaughanbrittonsage** [marcusolsson/grafana-gantt-panel#2]
Hi, I am unable to select any other field for start/end time than the configured @timestamp field for the datasource even though other date fields exist in the returned query data. I was wondering if this is supported for elasticsearch?

My data contains fields for:

```
{
    "@timestamp": .....,
    "StartTime": .........,
    "EndTime": .........
}
```

All are a datatype of `date` in the elasticsearch index. The start/end time fields are epoch millisecond precision timestamps but the data also contains other fields that are ISO datetime values and I get no option to select any other field. It only seems to allow me to select whatever field has been specified in the data source configuration as the timestamp field.

Thanks

**Comment by marcusolsson** [comment]
Thank you for reporting! It seems like some of the older data sources don't properly return time fields. I'll add the option to select string and number fields as well that will be converted. Stay tuned!

**Comment by marcusolsson** [comment]
I've published v0.2.0 which allows you to use string and number fields as time. Hopefully that will solve your issue, otherwise let me know!
